src/components/Input/Input.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+
+const Input = ({ id, title, visibility, value, onChange }) => (
+  <label htmlFor={id} className={visibility}>
+    {title}
```

Comment:
label? :)

Reply:
I don't catch this point? I put the name of the field in the "label" tag.